example-code/app/src/backend/RobotWorldRpcImpl.ts

```diff
@@ -2,12 +2,19 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-// __PUBLISH_EXTRACT_START__ RpcInterface.implementation
-import { IModelRpcProps, RpcInterface, RpcInterfaceDefinition } from "@bentley/imodeljs-common";
 import { Id64String } from "@bentley/bentleyjs-core";
+// __PUBLISH_EXTRACT_END__
```

Comment:
@paulius-valiunas any idea why this auto-formats to this have the _END_ here?  I tried to switch it locally and every time I saved the file it switched back this state.

Reply:
ah, I didn't notice that. It's the import sorter extension, it wants to have all imports in one place and treats those comments as attached to the imports.

Reply:
I changed it back to the way it was. We probably need to get rid of the sorter extension after all, I don't see a way to keep it from messing up files like this without completely disabling it.